iroh/src/magicsock.rs

```diff
@@ -2257,10 +2375,26 @@ impl Actor {
         }
 
         let mut receiver_closed = false;
+        #[cfg_attr(wasm_browser, allow(unused_mut))]
         let mut portmap_watcher_closed = false;
         let mut link_change_closed = false;
         loop {
             inc!(Metrics, actor_tick_main);
+            #[cfg(not(wasm_browser))]
+            let portmap_watcher_changed = portmap_watcher.changed();
+            #[cfg(wasm_browser)]
+            let portmap_watcher_changed = n0_future::future::pending();
+
+            #[cfg(not(wasm_browser))]
+            let direct_addr_heartbeat_timer_tick = direct_addr_heartbeat_timer.tick();
+            #[cfg(wasm_browser)]
+            let direct_addr_heartbeat_timer_tick = n0_future::future::pending();
+
+            #[cfg(not(wasm_browser))]
+            let link_change_r_recv = link_change_r.recv();
+            #[cfg(wasm_browser)]
+            let link_change_r_recv = n0_future::future::pending();
```

Comment:
not for this PR, but we should totally implement this, detecting connection changes seems to be sensible, even if we don't do direct connections

Reply:
Yepyep. Filed https://github.com/n0-computer/iroh/issues/3195 for this.